src/js/dataMapping/covid19/awardSpendingAgency/awardSpendingAgencyTableTabs.js

```diff
@@ -59,20 +59,20 @@ export const awardSpendingAgencyTableColumns = (type) => {
                     displayName: 'Agency Name'
                 },
                 {
-                    title: 'obligation',
-                    displayName: 'Award Obligations'
+                    title: 'faceValueOfLoan',
+                    displayName: 'Face Value of Loans'
                 },
                 {
-                    title: 'outlay',
-                    displayName: 'Award Outlays'
+                    title: 'obligation',
+                    displayName: 'Award Obligations (Loan Subsidy Cost)'
```

Comment:
One thing Marco and I discussed in our design review for #1787 was to force the line breaks of the column headings to match the mocks so they don't stack into too many rows and become hard to read. I've [opened a PR](https://github.com/fedspendingtransparency/data-transparency-ui/pull/46) to allow `displayName` to be an element or a string, and am using a combination of `whitespace: nowrap` and [two divs for the two lines of text](https://github.com/fedspendingtransparency/usaspending-website/pull/1787/files#diff-54ad7f6ed00c5d5c71bc26b013df8789R70) in the PR similar to this one for the loans table.